### DISCUSSION TITLE
json path 大数据量时 不好用 5万行数据

### DISCUSSION THREAD

**Discussion by hongjiea** [he3-app/he3-feedback#278] [Bug Report]
<img width="882" alt="image" src="https://github.com/he3-app/he3-feedback/assets/70626373/bc803782-922e-4a65-8b69-5a23747b88ad">
5 万行直接不好用，1 万行解析有误

**Comment by lyzhang1999** [comment]
感谢反馈，已创建 issues 跟进：https://github.com/he3-app/he3-feedback/issues/280
